c++/src/kj/async-coroutine-test.c++

```diff
@@ -286,7 +286,7 @@ KJ_TEST("Exceptions during suspended coroutine frame-unwind propagate via destru
   WaitScope waitScope(loop);
 
   auto exception = KJ_ASSERT_NONNULL(kj::runCatchingExceptions([&]() {
-    deferredThrowCoroutine(kj::NEVER_DONE);
+    auto promise = deferredThrowCoroutine(kj::NEVER_DONE);
```

Comment:
Can we use the old `(void)` syntax for these? In my mind it concisely says: "This function returns a value which I am intentionally ignoring." Assigning to a variable which isn't used seems more ambiguous.

```suggestion
    (void)deferredThrowCoroutine(kj::NEVER_DONE);
```

Reply:
Yeah, that's definitely better. Done.